scripts/aws/cfn/microservice.yml

```diff
@@ -440,25 +444,23 @@ Resources:
                 Effect: Allow
                 Resource: "*"
 
-  #BatchBridgeStateRule:
-  #  Type: AWS::Events::Rule
-  #  Properties:
-  #    Description: The EventBridge rule to match Batch state change
-  #    EventBusName: !Ref EventBusName
-  #    EventPattern:
-  #      source:
-  #        - aws.batch
-  #      detail-type:
-  #        - Batch Job State Change
-  #      detail:
-  #        status:
-  #          # - STARTED
-  #          # - SUCCEEDED
-  #          - FAILED
-  #    State: ENABLED
-  #    Targets:
-  #      - Arn: !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:${AlarmSNSTopicName}"
-  #        Id: !Ref EventBridgeName
+  ### Enable sending notifications to SNS topic when the job is in failed status.
+  BatchBridgeStateRule:
+   Type: AWS::Events::Rule
+   Properties:
+     Description: The EventBridge rule to match Batch state change
+     EventPattern:
+       source:
+         - aws.batch
+       detail-type:
+         - Batch Job State Change
+       detail:
+         status:
+           - FAILED
+     State: ENABLED
+     Targets:
+       - Arn: !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:${AlarmSNSTopicName}"
+         Id: !Sub "${ProjectName}-logsaver-be" 
```

Comment:
@danilosilvase what does "Id" represent?

Reply:
This ID is used to represent the target name within the specified rule. 

something like this:
![Screenshot 2022-10-26 at 11 47 54](https://user-images.githubusercontent.com/27283320/198007346-9d8f8b0b-d33e-4144-9343-ca1adf7f4a67.png)